.github/workflows/ci.yml

```diff
@@ -37,9 +37,6 @@ jobs:
 
           - operating-system: 'ubuntu-20.04'
             php-version: '7.4'
-            job-description: 'with calculating code coverage'
-            calculate-code-coverage: 'yes'
-            phpunit-flags: '--testsuite coverage --exclude-group covers-nothing --coverage-clover build/logs/clover.xml'
 
           - operating-system: 'ubuntu-20.04'
             php-version: '7.4'
```

Comment:
```suggestion
            php-version: '8.0'
```

Reply:
same few lines later for `job-description: 'with migration rules'`

Reply:
SRP, let's do code coverage PHP version update here and handle others in separate PR (as some syntax is broken with migration in PHP 8 and tests are failing).

Reply:
then drop the duplicated 7.4 job, we don't need 2 of them anymore